### PR TITLE
NFT rewards

### PR DESCRIPTION
**Pull request by whilefoo**

### CODE REVIEW

File: src/handlers/pricing/pricing-label.ts [hunk 1]
```diff
@@ -152,10 +152,14 @@ async function addPriceLabelToIssue(context: Context, targetPriceLabel: string,
...
-  });
-  return res.status === 200;
+  try {
+    await context.event.octokit.rest.issues.getLabel({
+      owner: payload.repository.owner.login,
+      repo: payload.repository.name,
+      name,
+    });
+    return true;
+  } catch (error) {
```
Comment (whilefoo) [inline src/handlers/pricing/pricing-label.ts:162]: octokit throws an error so this didn't work correctly

File: src/adapters/supabase/helpers/tables/wallet.ts [hunk 2]
```diff
@@ -167,6 +168,7 @@ export class Wallet extends Super {
     const { data: walletInsertData, error: walletInsertError } = await this.supabase
       .from("wallets")
       .insert(newWallet)
+      .select()
       .single();
```
Comment (whilefoo) [inline src/adapters/supabase/helpers/tables/wallet.ts:171]: there should be `select()` otherwise it doesn't work

**Review by 0x4007 — APPROVED** [review]
The merge target should be to the latest code (let me change this in a moment)

**Review by 0x4007 — APPROVED** [review]

**Comment by pavlovcik** [comment]
I assume this works so I'm merging it.

**Comment by 0x4007** [comment]
I didn't realize there are merge conflicts let me look into them.

**Pull request merged**